pathology/hovernet/infer_torch.py

```diff
@@ -0,0 +1,256 @@
+# Copyright 2020 MONAI Consortium
```

Comment:
I would say notebook for Infer is better and useful for people to try it out.
that is another way even someone can test/verify if e2e really works.